css-two-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+}
+
+header,
+footer {
+  text-align: center;
+}
+
+h1 {
+  font-family: karla, sans-serif;
+  font-weight: 700;
+}
+
+header > p {
+  font-family: karla, sans-serif;
+  font-weight: 400;
+  text-align: center;
+  font-size: small;
+}
+
+.container {
```

Comment:
The container should have a set width on it, for this exercise I would recommend 1200px. Its good to remember to set hard coded values like pixels on the container, and rows and columns should be percentages

Reply:
Noted.